lib/core/test/unit/Cardano/Wallet/Primitive/ModelSpec.hs

```diff
@@ -788,8 +796,8 @@ prop_applyOurTxToUTxO_noneOurs slotNo blockHeight tx utxo =
     report (utxoFromTx tx) "utxoFromTx tx" $
     property $ utxo' `isSubsetOf` utxo
   where
-    utxo' =
-        maybe utxo snd $ applyOurTxToUTxO slotNo blockHeight NoneOurs tx utxo
+    utxo' = maybe utxo snd $
+        applyOurTxToUTxO slotNo blockHeight (OurUtxo utxo) tx utxo
```

Comment:
Thanks, I like this new type much better. 😊 Though now the property could, in principle, be violated, because the transaction `tx` may contain outputs with addresses that are also in the `utxo`.